### PR TITLE
remove OMSSA

### PR DESCRIPTION
**Pull request by timosachsenberg**
# Description

Depends on https://github.com/OpenMS/THIRDPARTY/pull/75
TODO: update tutorials that still use OMSSA

# Checklist:
- [X] Make sure that you are listed in the AUTHORS file
- [X] Add relevant changes and new features to the CHANGELOG file
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] New and existing unit tests pass locally with my changes
- [X] Updated or added python bindings for changed or new classes. (Tick if no updates were necessary.)

# How can I get additional information on failed tests during CI:
If your PR is failing you can check out 
- http://cdash.openms.de/index.php?project=OpenMS and look for your PR. If you click in the column that lists the failed tests you will get detailed error messages.

### CODE REVIEW

**Comment by timosachsenberg** [comment]
failed test unrelated

File: share/OpenMS/examples/TOPPAS/Ecoli_Identification.toppas [hunk 1]
```diff
@@ -140,7 +140,7 @@
     <NODE name="2" description="">
       <ITEM name="recycle_output" value="false" type="string" description="" required="false" advanced="false" />
       <ITEM name="toppas_type" value="tool" type="string" description="" required="false" advanced="false" />
-      <ITEM name="tool_name" value="OMSSAAdapter" type="string" description="" required="false" advanced="false" />
+      <ITEM name="tool_name" value="TODO" type="string" description="" required="false" advanced="false" />
```
Comment (oliveralka) [inline share/OpenMS/examples/TOPPAS/Ecoli_Identification.toppas:143]: Could we add CometAdapter here, or does the toppas workflow would have to be redone?

**Review by oliveralka — COMMENTED** [review]
Looks good!

**Review by oliveralka — APPROVED** [review]

File: share/OpenMS/examples/TOPPAS/Ecoli_Identification.toppas (hunk 1 above)
Comment (timosachsenberg) [reply share/OpenMS/examples/TOPPAS/Ecoli_Identification.toppas:143]: needs to be redone

Comment (timosachsenberg) [reply share/OpenMS/examples/TOPPAS/Ecoli_Identification.toppas:143]: will do now

**Comment by jpfeuffer** [comment]
shouldnt there be stuff in IDPEP and ConsensusID?

**Comment by timosachsenberg** [comment]
yes but this is deliberately kept so old results can still be processed.

**Comment by jpfeuffer** [comment]
ah yes, ok.

**Comment by cbielow** [comment]
a bit late to the party but: in my benchmarks, OMSSA was usually doing a very decent job, especially when it comes to tricky searches.
I can imagine (no data to prove it), that it adds value when used in ConsensusID & Co?
So I'm not really sure it's warranted to remove it?
The fact that its not actively maintained is unfortunate, but is there anything 'broken' which prevents people from using it?

**Comment by timosachsenberg** [comment]
Did not really add value to ConsensusID in our experiments and with Comet, X!Tandem and MSGFPlus there was no real benefit and so many other more advanced alternatives ready. It also was removed from conda afaik.

**Comment by cbielow** [comment]
ok.

**Pull request merged**